cmd/chart.go

```diff
@@ -94,23 +98,25 @@ func moveChart(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to parse output flag: %w", err)
 	}
 
-	chartMover, err := mover.NewChartMover(
-		&mover.ChartMoveRequest{
-			Source: mover.Source{
-				Chart: mover.ChartSpec{
-					Local: mover.LocalChart{Path: args[0]},
-				},
-				ImageHintsFile: imagePatternsFile,
-			},
-			Target: mover.Target{
-				Chart: mover.ChartSpec{
-					Local: mover.LocalChart{Path: outputPathFmt},
-				},
-				Rules: *targetRewriteRules,
+	moveRequest := mover.ChartMoveRequest{
+		Source: mover.Source{
+			Chart: mover.ChartSpec{
+				Local: &mover.LocalChart{Path: args[0]},
 			},
+			ImageHintsFile: imagePatternsFile,
 		},
-		mover.WithRetries(retries), mover.WithLogger(cmd),
-	)
+		Target: mover.Target{
+			Chart: mover.ChartSpec{},
+			Rules: *targetRewriteRules,
+		},
+	}
+	if toArchive != "" {
```

Comment:
in the future we should probably validate that either of this options (not both of them) are set.